Mathlib/Algebra/CharP/Basic.lean

```diff
@@ -54,92 +54,91 @@ end Commute
 
 section CommSemiring
 
-variable [CommSemiring R] {p : ℕ} {x y : R}
+variable [CommSemiring R] {p : ℕ} (x y : R) (n : ℕ)
```

Comment:
```suggestion
variable [CommSemiring R] {p : ℕ} (hp : p.Prime) (x y : R) (n : ℕ)
include hp
```
so that `hp` stays before `x y n`

Reply:
Oh yes, thank you, `include hp` works well. I also changed the section above.